Created plugin for LUADNS

albertofustinoni opened the pull request:
Created DNS record verification plugin for [LUADNS](https://api.luadns.com/) using their [REST API](http://www.luadns.com/api.html)

WouterTinus commented on src/plugin.validation.dns.luadns/luadns.cs:
```diff
@@ -0,0 +1,141 @@
+
+                var payload = await response.Content.ReadAsStringAsync();
+                var zones = JsonSerializer.Deserialize<ZoneData[]>(payload);
+                var targetZone = zones.Where(d => recordName.EndsWith(d.Name)).OrderByDescending(d => d.Name.Length).FirstOrDefault();
```
You may want to consider handling case sensitivity here.

albertofustinoni commented:
Hello @WouterTinus 

I have created a new pull request #1514  for branch 2.1.7 (this is what you meant by rebasing?) and addressed your suggestions as well

WouterTinus commented:
Closing this because the other one was merged.
